### PR TITLE
[CLI-216] fix the services removal by calling apis sequentially

### PR DESCRIPTION
**Pull request by codemodify**
### Description
Calling APIs sequentially
Progress indicator was changed to have the same nr or characters to avoid display issues
Changed the UDP buffer size for client/server to accommodate the status command to be able to send/receive 30+ services otherwise it will error-out with JSON breaking

### Checklist
- [x] Pull Request title is in the format `[PROJ-12345] Some useful description here...`
- [ ] ~~I have prefixed this PR with `[WIP]` if it is a Work In Progress (not ready to merge)~~
- [x] The git branch is in the format `PROJ-12345-some-useful-description-here`
- [x] I have added one or more reviewers
- [ ] ~~I have added/updated tests for any code changes~~
- [ ] ~~I have updated documentation (including README, inline comments and docs.remote.it docs)~~
- [x] This PR only contains one isolated change (bug fix, feature, etc)
- [ ] ~~Translation strings added/updated for all user-visible text~~
- [x] I have manually reviewed this PR to make sure only the files I intended to change were changed and nothing else
- [x] I have notified reviewers on Slack by @ mentioning them in the `#cli`channel
- [x] I have moved the Jira ticket into `Resolved` state and made a note in the ticket with a link to this PR

### Ticket
https://remoteit.atlassian.net/browse/CLI-216

### CODE REVIEW

File: schannel/schannel.go [hunk 1]
```diff
@@ -82,7 +82,7 @@ func (s *Server) Listen() {
 		logger.Debugf("server listening %s:%d", s.IP, s.Port)
 
 		for {
-			message := make([]byte, 4096)
+			message := make([]byte, 1048576) // 1MB
```
Comment (danawoodman) [inline schannel/schannel.go]: Please move this value to globals, mabye `UDP_BUFFER_SIZE` or something like that

File: schannel/client.go [hunk 2]
```diff
@@ -75,7 +75,7 @@ func (c *Client) Send(msg string) (response []byte, err error) {
 
 		logger.Debugf("send: sent message")
 
-		resp := make([]byte, 2048)
+		resp := make([]byte, 1048576) // 1MB
```
Comment (danawoodman) [inline schannel/client.go]: Make global (see below)

File: schannel/schannel.go (hunk 1 above)
Comment (codemodify) [reply schannel/schannel.go]: fixed

File: schannel/client.go (hunk 2 above)
Comment (codemodify) [reply schannel/client.go]: fixed

**Comment by codemodify** [comment]
> I think we want to call the device delete API instead correct?

fixed

**Pull request merged**